manage

```diff
@@ -20,6 +20,8 @@ source "$(dirname "${BASH_SOURCE[0]}")/utils/lib_go.sh"
 # shellcheck source=utils/lib_redis.sh
 source "$(dirname "${BASH_SOURCE[0]}")/utils/lib_redis.sh"
 
+PATH="${REPO_ROOT}/node_modules/.bin;${PATH}"
```

Comment:
Ugly typo in path separator from mine:  `;` --> `:`

Reply:
Damn that's a nasty typo! Was that the cause of your NVM problems?

Reply:
yes

Reply:
Well, I'm glad that it wasn't anything more complicated!